build.sh

```diff
@@ -1,7 +1,8 @@
 #!/bin/bash
-
-
 set -e
 echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
-docker build -t $DOCKER_ORG/init-controller:$TRAVIS_TAG -t $DOCKER_ORG/init-controller:latest .
-docker push $DOCKER_ORG/init-controller
+export DOCKER_CLI_EXPERIMENTAL=enabled
+docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
+docker buildx create --name mybuilder
+docker buildx use mybuilder
+docker buildx build -t appsody/init-controller:$TRAVIS_TAG -t appsody/init-controller:latest --platform=linux/amd64,linux/ppc64le,linux/s390x . --push
```

Comment:
I see we replaced `$DOCKER_ORG` with `appsody`, I think it's fine. Just wondering if it was intentional?

Reply:
Ahh we should keep docker_org for people making their own custom controllers, my bad